datasets/ilist/README.md

```diff
@@ -0,0 +1,165 @@
+---
+task_categories:
+- text-classification
+multilinguality:
+- multilingual
+task_ids:
+- other
```

Comment:
`multi-class-classification` maybe ? @lhoestq WDYT ?

Reply:
I would say language identification.
You can use
```
task_ids:
- text-classification-other-language-identification
- ```